pkgs/applications/science/electronics/gtkwave/default.nix

```diff
@@ -23,7 +24,13 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkg-config wrapGAppsHook ];
-  buildInputs = [ bzip2 glib gperf gtk3 judy tcl tk xz ];
+  buildInputs = [ bzip2 glib gperf gtk3 judy tcl tk xz ]
+    ++ lib.optional stdenv.isDarwin gtk-mac-integration;
+
+  patches = [ ] ++ lib.optional stdenv.isDarwin [
+    ./0001-Fix-detection-of-quartz-in-gdk-3.0-target.patch
```

Comment:
Please use fetchpatch, and make the patches unconditional.

Reply:
The pr commit applies to the master branch, but it cannot apply to the release tarball.